test/e2e/ipsec.go

```diff
@@ -80,7 +80,7 @@ var _ = ginkgo.Describe("IPSec", func() {
 				gomega.Expect(err).Should(gomega.BeNil())
 				gomega.Expect(int(ipsecMetricValue)).Should(gomega.Equal(1))
 
-				ginkgo.By(fmt.Sprintf("Verify ipsec tunnel metrics reflecting up from ovn pod %s. ", ovnPod.Name))
+				ginkgo.By(fmt.Sprintf("Verify ipsec tunnel metrics reflecting up from ovn pod %s ", ovnPod.Name))
```

Comment:
nit: remove trailing space in the log message.

Reply:
done